### PR TITLE
Ks introduce pickup requests

### PR DESCRIPTION
**Pull request by skorbut**
Fixed typo in readme,
Added segment describing pickup request creation,
Added spec and implemention to request pickup through shipcloud

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,28 @@ shipment_quote = Shipcloud::ShipmentQuote.create(
 shipment_quote.price # => 6.2
 ```
 
+### Create a pickup request
+
+To request parcels being picked up, you need to provide the carrier name and the time (earliest and latest) your shipments shall be fetched. You may also provide a list of shipment ids to specify only certain shipments to be included in the pickup request.
```
Comment (jj3l) [inline README.md]: "carrier id" would be more precise than "carrier name".

Comment (jj3l) [inline README.md]: Should we describe the default behaviour here providing no shipments ids?

Comment (skorbut) [inline README.md]: Using "name" is consistent with the rest of the readme. Shall I change the other occurences as well?
I would rather document the more complete form, specifying the format the shipment list is to be built.

Comment (jj3l) [inline README.md]: OK consistence over naming. I think "id" is more precise than "name" but this might be an issue for another pr.

As a user of the API I should know the list of shipments may be empty.

**Comment by jj3l** [comment]
Ready to merge after squash.
